claragenomics/dl4atac/models/model_utils.py

```diff
@@ -0,0 +1,133 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+import configargparse
+
+# system imports
+import logging
+import os
+import random
+import sys
+import tempfile
+import time
+
+import h5py
+import multiprocessing
+import numpy as np
+import pandas as pd
+import torch
+import torch.backends.cudnn as cudnn
+import torch.distributed as dist
+import torch.multiprocessing as mp
+import torch.nn as nn
+from torch.nn.parallel import DistributedDataParallel
+from torch.optim import Adam
+
+# module imports
+from claragenomics.dl4atac.models.models import *
+from claragenomics.dl4atac.utils import *
+
+# python imports
+import warnings
+import glob
+import math
+warnings.filterwarnings("ignore")
+
+# Set up logging
+log_formatter = logging.Formatter(
+    '%(levelname)s:%(asctime)s:%(name)s] %(message)s')
+_logger = logging.getLogger('AtacWorks-main')
+_handler = logging.StreamHandler()
+_handler.setLevel(logging.INFO)
+_handler.setFormatter(log_formatter)
+_logger.setLevel(logging.INFO)
+_logger.addHandler(_handler)
+
+
+def model_args_v1():
+    parser = configargparse.ArgParser(default_config_files=['configs/model_structure.yaml'])
```

Comment:
let's pas sin the absolute path to the default file

Reply:
done.